Documentation/ApiOverview/DirectoryStructure/Index.rst

```diff
@@ -21,18 +21,76 @@ almost always find the directory structure as outlined below.
 Also see :ref:`Environment` for further information, especially how to retrieve
 the paths within PHP code.
 
+Files on project level
+======================
+
+On the top most level, the project level, you can find the files
+:file:`composer.json` which contains requirements for the TYPO3 installation
+and the :file:`composer.lock` which contains information about the concretely
+installed versions of each package.
+
 Directories in a typical project
 ================================
 
 .. contents::
    :local:
 
+.. _directory-config:
+
+:file:`config/`
+---------------
+
+Composer installations only. TYPO3 configuration directory. This directory
+contains installation-wide configuration.
+
+.. _directory-config-sites:
+
+:file:`config/sites/`
+~~~~~~~~~~~~~~~~~~~~~
+
+Composer installations only.  The folder :file:`config/sites` contains
+subfolders for each :ref:`site configuration <sitehandling>`.
+
+
+.. _directory-local_packages:
+
+:file:`local_packages/`
```

Comment:
When you introduce now a "local_packages" folder, please take care of other places in the TYPO3 docs, where "packages" is used (to be consistent), for example here: https://docs.typo3.org/m/typo3/guide-installation/main/en-us/MigrateToComposer/BestPractices.html#define-a-local-path-repository
I assume, there are other places.

Reply:
We already use local_packages in the sitepackage and would take care of this in a follow up